### PR TITLE
fix(radios): add missing demo page <hx-radio-set> elements

### PR DESCRIPTION
**Pull request by 100stacks**
### Free Friday

* add missing `<hx-radio-set>` elements to the following demo pages:
  * box
  * button
  * file
  * modal
  * table

### LGTM's
- [x] Dev LGTM
- [x] Zoom LGTM

### CODE REVIEW

**Comment by HelixUI** [comment]
Deploy preview for *helix-ui* processing.

Building with commit 579927f54ba6dd883031f62865efa7128d96465f

https://app.netlify.com/sites/helix-ui/deploys/5d169a5649f2ee00075ef7a0

File: docs/components/table/index.html [hunk 1]
```diff
@@ -20,33 +20,35 @@ <h2 id="basic-table">Basic Table</h2>
...
-                Condensed
-              </label>
-            </hx-radio-control>
+              <hx-radio-control>
+                <input
+                  id="radCondensed"
+                  name="density"
+                  type="radio"
+                  v-model="isCondensed"
+                  :value="true"
```
Comment (cathysiller) [inline docs/components/table/index.html]: can we pls move `:value="true"` above `id="radCondensed"`? seems like we're going with this pattern in this p/r

File: docs/components/table/index.html [hunk 2]
```diff
@@ -20,33 +20,35 @@ <h2 id="basic-table">Basic Table</h2>
...
-              </label>
-            </hx-radio-control>
+            <hx-radio-set>
+              <hx-radio-control>
+                <input
+                  id="radNormal"
+                  name="density"
+                  type="radio"
+                  v-model="isCondensed"
+                  :value="false"
```
Comment (cathysiller) [inline docs/components/table/index.html]: can you move `:value="false"` above `id="radNormal"`? seems like we're going with this pattern in this p/r

File: docs/components/table/index.html (hunk 1 above)
Comment (100stacks) [reply docs/components/table/index.html]: moved `:value="true"` above `id="radCondensed"`

File: docs/components/table/index.html (hunk 2 above)
Comment (100stacks) [reply docs/components/table/index.html]: moved `:value="false"` above `id="radNormal"`

**Comment by 100stacks** [comment]
@CITguy, submitted update to address Comment https://github.com/rackerlabs/helix-ui/pull/521#pullrequestreview-256590307

File: src/helix-ui/styles/components/radios.less [hunk 3]
```diff
@@ -13,6 +13,28 @@ hx-radio {
...
+    &:first-child {
+      margin-top: 0; // FIXME: solve with spacing system
+    }
+
+    &:last-child {
+      margin-bottom: 0; // FIXME: solve with spacing system
+    }
+  }
+
+  > legend {
```
Comment (CITguy) [inline src/helix-ui/styles/components/radios.less]: Remove `> legend` block

File: src/helix-ui/styles/components/radios.less [hunk 4]
```diff
@@ -13,6 +13,28 @@ hx-radio {
...
+    color: @gray-800;
+    font-size: 0.875rem;
+  }
+
+  > label {
+    display: block;
+    font-size: 1rem;
+  }
```
Comment (CITguy) [inline src/helix-ui/styles/components/radios.less]: Remove `> label` block.

File: src/helix-ui/styles/components/radios.less (hunk 3 above)
Comment (100stacks) [reply src/helix-ui/styles/components/radios.less]: `> legend` block removed

File: src/helix-ui/styles/components/radios.less (hunk 4 above)
Comment (100stacks) [reply src/helix-ui/styles/components/radios.less]: `> label` block removed

**Pull request merged**